server/db/config/build.js

```diff
@@ -5,7 +5,8 @@ const connection = require('./connection');
 
 const dbBuild = () => {
   const sql = readFileSync(join(__dirname, 'build.sql')).toString();
-  return connection.query(sql);
+  return connection.query(sql).then(console.log('Done')).catch((err => console.log(err)));
```

Comment:
catch errors when you call the function
```suggestion
  return connection.query(sql);
```